### ISSUE TITLE
Am  I doing something wrong or should I put in a bug request??  Hangs for over 5 minutes after putting in command line

### ISSUE THREAD

**Issue by shellig** [yt-dlp/yt-dlp#2666]
### Checklist

- [X] I'm asking a question and **not** reporting a bug/feature request
- [X] I've looked through the [README](https://github.com/yt-dlp/yt-dlp#readme)
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar questions including closed ones

### Question

It takes about 5-6 minutes before anything happens.  I didn't start recording the attached video for almost a minute before I thought of it....
It was taking about 30 seconds before - nothing changed on my computer other than downloading the new version to help with throttling issue.

[https://1drv.ms/v/s!Aj0Dz8zBtvvEhqVAjllP-ECL0BmLwA](url)  Would not let me attach - sharing  the one on my OneDrive - really boring, though just showing the lag.  I click in the window and hit space or enter multiple times - it is not that.

### Verbose log

```shell
W:\Media\Videos\YouTube>yt-dlp -vu -4 -w -f "135/136/22/mp4" -o "W:\Media\Videos\YouTubeCourses\%%(upload_date)s %%(uploader_id)s %%(title)s.%%(ext)s" https://www.youtube.com/watch?v=v2oNWja7M2E
[debug] Command-line config: ['-vu', '-4', '-w', '-f', '135/136/22/mp4', '-o', 'W:\\Media\\Videos\\YouTubeCourses\\%%(upload_date)s %%(uploader_id)s %%(title)s.%%(ext)s', 'https://www.youtube.com/watch?v=v2oNWja7M2E']
Type account password and press [Return]:
[debug] Encodings: locale cp1252, fs utf-8, out utf-8, err utf-8, pref cp1252
[debug] yt-dlp version 2022.02.04 [c1653e9] (win_exe)
[debug] Python version 3.8.10 (CPython 64bit) - Windows-10-10.0.19043-SP0
[debug] exe versions: ffmpeg N-80011-gaf3e944 (needs_adtstoasc)
[debug] Optional libraries: Cryptodome, mutagen, sqlite, websockets
[debug] Proxy map: {}
WARNING: [youtube] Cannot login to YouTube using username and password. Use --cookies-from-browser or --cookies for the authentication. See  https://github.com/ytdl-org/youtube-dl#how-do-i-pass-cookies-to-youtube-dl  for how to manually pass cookies
[debug] [youtube] Extracting URL: https://www.youtube.com/watch?v=v2oNWja7M2E
[youtube] v2oNWja7M2E: Downloading webpage
[youtube] v2oNWja7M2E: Downloading android player API JSON
[debug] Sort order given by extractor: quality, res, fps, hdr:12, source, codec:vp9.2, lang, proto
[debug] Formats sorted by: hasvid, ie_pref, quality, res, fps, hdr:12(7), source, vcodec:vp9.2(10), acodec, lang, proto, filesize, fs_approx, tbr, vbr, abr, asr, vext, aext, hasaud, id
[info] v2oNWja7M2E: Downloading 1 format(s): 135
[debug] Invoking downloader on "https://rr5---sn-5ualdn76.googlevideo.com/videoplayback?expire=1644222458&ei=moMAYtT-E-uAlAOh7bGAAg&ip=73.184.84.133&id=o-AHZAwtMTjiHy846qd6euvxhwdgIHGRwSPKPDEyjp1yVF&itag=135&source=youtube&requiressl=yes&mh=W2&mm=31%2C26&mn=sn-5ualdn76%2Csn-p5qlsnd6&ms=au%2Conr&mv=m&mvi=5&pl=21&initcwndbps=2037500&vprv=1&mime=video%2Fmp4&gir=yes&clen=31057277&dur=2955.599&lmt=1628765641494268&mt=1644200462&fvip=5&keepalive=yes&fexp=24001373%2C24007246&c=ANDROID&txp=5432434&sparams=expire%2Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cvprv%2Cmime%2Cgir%2Cclen%2Cdur%2Clmt&sig=AOq0QJ8wRgIhAJuAhmZPdbWA3NSiUdVdD9Fb1fkGRoV83S0n9GhkFfGLAiEA1veY7cFJgGxywBwYYoWTR0oSj6iQQ_GbQMsB3jWicw0%3D&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps&lsig=AG3C_xAwRQIgMct5ezfsrVaoo3ms0caYdJEx6ouVEs03ko5NAJ2l3b4CIQC3XU0PTsemnfsoP6DUf1IVnld8jkW7GVjCjOjrHfBUfQ%3D%3D"
[download] Destination: W:\Media\Videos\YouTubeCourses\%(upload_date)s %(uploader_id)s %(title)s.%(ext)s
[download] 100% of 29.62MiB in 00:28
```

**Comment by pukkandan** [comment]
Most likely your antivirus is causing the slowdown. try running after disabling it

**Comment by shellig** [comment]
Yes, seems to have helped both initiation and download speed, THANKS!

**Comment by shellig** [comment]
Thanks. Did not get audio with that command so trying to figure that out now. :(

Sent from my iPad

On Feb 6, 2022, at 23:20, pukkandan ***@***.***> wrote:

﻿

Closed #2666<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fyt-dlp%2Fyt-dlp%2Fissues%2F2666&data=04%7C01%7C%7C1cfd81c95d5e438481d008d9e9f126df%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637798044213937443%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=Qup1xdc7FCaTWXfxdYR5fr5YUrK2KsiYsy9UFNSyjdE%3D&reserved=0>.

—
Reply to this email directly, view it on GitHub<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fyt-dlp%2Fyt-dlp%2Fissues%2F2666%23event-6016387113&data=04%7C01%7C%7C1cfd81c95d5e438481d008d9e9f126df%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637798044213937443%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=J0pMEO2Wqksreao%2BdI3EDbU37JHstFApF5e1Q7v7Aoc%3D&reserved=0>, or unsubscribe<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAEW67GO4GSLMYY5FB5B4LSLUZ5CAHANCNFSM5NWG7JKQ&data=04%7C01%7C%7C1cfd81c95d5e438481d008d9e9f126df%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637798044213937443%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=gqD9%2Bq1Wt%2BCEIdVR%2BRo50eVQpvPD2EmRIZUqdRCBI0s%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7C%7C1cfd81c95d5e438481d008d9e9f126df%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637798044213937443%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=FoQJQF77zbuIxxFipImE34616JIArwYtI0aMTjEc%2F54%3D&reserved=0> or Android<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26referrer%3Dutm_campaign%253Dnotification-email%2526utm_medium%253Demail%2526utm_source%253Dgithub&data=04%7C01%7C%7C1cfd81c95d5e438481d008d9e9f126df%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637798044213937443%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=0oEdkBVvaU%2BzM6va30UT5zWLTtqD2WfxDpyGbyaXQl0%3D&reserved=0>.
